server/models/UserModel/index.js

```diff
@@ -0,0 +1,48 @@
+import config from '../../configs/config'
+import { collections } from '../../utils/mongoCollections'
+import { USER_MONGO_PROJECTION } from '../../constants'
+
+const UserModel = {
+  withDefaults: (overrides = {}) => ({
+    display_name: '',
+    title: '',
+    department: '',
+    company: '',
+    mail: '',
+    member_of: '',
+    bad_pwd_count: 0,
+    lockout_time: 0,
+    last_logoff: Date.now(),
+    last_logon: Date.now(),
+    account_expires: null,
+    password: '',
+    ldap: false,
+    force_reset_pw: false,
+    realms: config.app.realms,
+    icon: '',
+    access: [],
+    blocked: false,
+    role: 'member',
+    ...overrides,
+  }),
+  save: async (db, userAttributes) => {
+    const newUser = UserModel.withDefaults(userAttributes)
+
+    return await db.collection(collections.users).insertOne(newUser)
+  },
+  findUserAndUpdate: async (db, userQuery, userUpdates) => {
+    return await db.collection(collections.users).findOneAndUpdate(
+      userQuery,
+      {
+        $set: userUpdates,
+      },
+      {
+        projection: USER_MONGO_PROJECTION,
```

Comment:
this is now the only place where the `USER_MONGO_PROJECTION` is used so I think we can move the constant into this file and no longer export it.

Reply:
[1dfc359](https://github.com/AMP-SCZ/dpdash/pull/293/commits/1dfc359bef2eb895ab0c40a03cfe39d9024cdc53)